config/deploy.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+set :application, 'was-pywb'
+set :repo_url, "https://github.com/sul-dlss/#{fetch(:application)}.git"
+
+# Default branch is :master
+ask :branch, `git rev-parse --abbrev-ref HEAD`.chomp
+
+# Default deploy_to directory is /var/www/my_app_name
+set :deploy_to, '/home/was/swap'
+
+# Default value for :format is :airbrussh.
+# set :format, :airbrussh
+
+# You can configure the Airbrussh format using :format_options.
+# These are the defaults.
+# set :format_options, command_output: true, log_file: "log/capistrano.log", color: :auto, truncate: :auto
+
+# Default value for :pty is false
+# set :pty, true
+
+# Default value for :linked_files is []
+# append :linked_files, "config/database.yml", 'config/master.key'
+
+# Default value for linked_dirs is []
+# append :linked_dirs, "log", "tmp/pids", "tmp/cache", "tmp/sockets", "tmp/webpacker", "public/system", "vendor"
+
+# Default value for default_env is {}
+# set :default_env, { path: "/opt/ruby/bin:$PATH" }
+
+# Default value for local_user is ENV['USER']
+# set :local_user, -> { `git config user.name`.chomp }
+
+# Default value for keep_releases is 5
+# set :keep_releases, 5
+
+# Uncomment the following to require manually verifying the host key before first deploy.
+# set :ssh_options, verify_host_key: :secure
+
+after 'deploy:publishing', 'uwsgi:restart' do
+  on roles(:app) do
+    within "#{current_path}/pywb" do
+      execute :pip3, :install, '-r requirements.txt'
+      execute :uwsgi, 'uwsgi.ini'
+    end
+  end
+end
```

Comment:
Here's the actual deploy. This hook is part of both the deployment and rollback flows. We need to replace line 44 with something systemd based (see #18), though, because right now the deployment never exists since that command is blocking. 😆